effectsize_example.Rmd

```diff
@@ -85,7 +100,7 @@ library(ggstance)   # for geom_pointrangeh()
 
 ```{r boilerplate, include = FALSE}
 set.seed(12)
-format_num <- function(...) format(..., digits = 2)
+format_num <- function(x, k = 2) format(round(x, k), nsmall = k)
```

Comment:
Re: significant figures I think the effect size FAQ is not the place for that discussion, but rather a separate document that talks about how we report numbers across the entire set of guidelines. Making an issue for that.